block.json

```diff
@@ -45,9 +48,8 @@
 	"supports": {
 		"html": false
 	},
-	"textdomain": "winamp-block",
 	"editorScript": "file:./build/index.js",
 	"editorStyle": "file:./build/index.css",
-	"script": "winamp-block-frontend",
+	"script": "winamp-block-frontend.js",
```

Comment:
this will not work as the `winamp-block-frontend` string refers to the handle the script was registered under.